cli/test/fixtures/seo/seo-failure-cases.html

```diff
@@ -5,7 +5,7 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 -->
 
-<html>
+<html lang="en">
```

Comment:
It'd be good to update this smoke test to include cases that exercise the new logic (`getLangOfInnerText`). The expectations for this smoke tests are found in `seo-failing.js`. If the current smoke expectations were more explicit,  it would have this for the `link-text` audit (currently it just checks for 4 failures, but it'd be good to be more explicit now):

```json
'link-text': {
  score: 0,
  displayValue: '4 links found',
  details: {
    items: [
      {text: 'click this'},
      {text: 'click this'},
      {text: 'CLICK THIS'},
      {text: 'click this'},
    ],
  },
}
```

Can you devise some cases here to make sure `getLangOfInnerText` works as expected? If you need to create a new file HTML in `fixtures/seo` and a new accompanying expectation file in `test-definitions/seo-mixed-language.js`, that's OK.

Reply:
I changed the smoke test `seo-failing` to be more explicit, including the language attribute and a German example. I also added a new smoke test `seo-mixed-language` with many edge cases (German and English examples).